spec/models/movie_spec.rb

```diff
@@ -4,4 +4,16 @@
   describe 'relationships' do
     it {is_expected.to have_many(:actors).through(:actor_movies)}
   end
+
+  describe 'activerecord' do
+    it 'returns average rating of movies' do
+      director = Director.create!(name: 'Wakowski Sisters')
+      movie1 = Movie.create!(director_id: director.id, title: "Guardians of the Galaxy", description: "Practically perfect in every way", rating: 5)
+      movie2 = Movie.create!(director_id: director.id, title: "Suicide Squad", description: "Practically perfect in every way", rating: 15)
+
+      average = (movie1.rating + movie2.rating) / 2
```

Comment:
yes, but we wanted the average of the movies within a genre, so this should have been in your models/genre_spec.rb instead, with fuller testing around movies in multiple genres to ensure the average was calculated properly.